### ISSUE TITLE
idea

### ISSUE THREAD

**Issue by ZSaama** [LeaPhant/skyblock-stats#7]
i dont know if it is possible to show the fairy souls that are missing

**Comment by LeaPhant** [comment]
Nope, unfortunately not. The API only returns the amount of fairy souls you've already found.
